src/test/java/org/example/jakarta_labb/resource/MovieResourceTest.java

```diff
@@ -0,0 +1,97 @@
+package org.example.jakarta_labb.resource;
+
+import org.example.jakarta_labb.dto.MovieDto;
+import org.example.jakarta_labb.entity.Movie;
+import org.example.jakarta_labb.repository.MovieRepository;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import java.util.List;
+import java.util.UUID;
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.*;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+
+
+@ExtendWith(MockitoExtension.class)
+public class MovieResourceTest {
+
+    @Mock
+    private MovieRepository movieRepository;
+
+    @InjectMocks
+    private MovieResource movieResource;
```

Comment:
If you only want to mock the MovieService you could remove the "private" key word in MovieResource. That way the field would be accessable in this test. https://github.com/Lucia560/Jakarta_labb/blob/main/src/main/java/org/example/jakarta_labb/resource/MovieResource.java#L18

A little something moving forward with the new set of tests for the endpoints. All these tests are redundant since they are tested in movie service test.

Reply:
Angela, you are an 😇. Guys, this code I wrote before the service class and a few other additions were done. As I got sick for the last days, and panicked, I did not fetch in this branch the last changes, so when the tests passed locally I just pushed it. I redo the tests now as this are, as Angela says, redundant. They dont test the endpoints anyway.